04.android/06.solving-problems/15.reinstallation issue when using multiple profiles on Android devices/docs.en.md

```diff
@@ -0,0 +1,19 @@
+---
+title: 'Problem with reinstalling AdGuard Ad Blocker on Android devices when using the app on multiple accounts'
+taxonomy:
+    category:
+        - docs
+visible: true
+---
+
+The owners of Android 9 and up tablets or smartphones with additional accounts setted might face the problem with AdGuard Ad Blocker removal. If AdGuard is installed on multiple accounts, when you try to delete it from one account, the app remains in the list of applications, but you can’t do anything with it — uninstall or reinstall. This happens because AdGuard is installed in a different user account on the device. 
+
+If you try to reinstall AdGuard AdBlocker after a failed attempt of removal, you will see the error "You can't install the app on your device".
+
+To solve this problem you need to delete the application for all users: go to Settings -> All apps -> AdGuard. Tap three dots at the upper right corner and then tap *Uninstall for all users*.
+
+<img src="https://cdn.adguard.com/public/Adguard/kb/android/multiple_users/uninst_en.png" style="border: 1px solid #efefef; max-width: 350px; padding: 2px;">
```

Comment:
Do not specify additional styles for images.

The images markup should be handled by the website's CSS styles instead.
If you want to limit the maximum width, consider resizing the image to the desired max width.